spec/models/course/assessment_spec.rb

```diff
@@ -305,16 +302,6 @@
         submissions = assessments.flat_map(&:submissions)
         expect(submissions.all? { |submission| submission.course_user.user == user1 }).to be(true)
       end
-
-      it 'returns submissions in reverse chronological order' do
```

Comment:
This should not be removed. Instead, you should create 2 assessments and 2 submissions.